puppet/modules/slave/templates/test_pull_requests_foreman_digitalocean.json.erb

```diff
@@ -0,0 +1,49 @@
+{
+    "proxy_host": null,
+    "proxy_port": null,
+    "jenkins_host": "http://ci.theforeman.org/",
+    "jenkins_url": "http://ci.theforeman.org/",
+    "hub_config": "~/.config/hub",
+    "github_user": "theforeman",
+    "bot_github_user": "<%= github_user -%>",
+    "irc_channel": "#theforeman-dev",
+    "not_mergable_cache": "~/pr_tests/cache/test_pull_request_foreman_digitalocean_not_mergable",
+    "repo_groups": {
+        "foreman-digitalocean": "foreman-digitalocean"
+    },
+    "settings": {
+        "test_settings": {
+            "name": "test",
+            "test_prefix": "Test Results:",
+            "allow_multiple": true,
+            "run_without_trigger": true,
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_plugin_pull_request",
+                    "downstream_job_name": "test_plugin_foreman-digitalocean_master",
+                    "build_token": "<%= jenkins_build_token -%>",
+                    "addtl_jenkins_params": {
+                        "foreman-digitalocean": {
+                            "plugin_repo": "https://github.com/theforeman/foreman-digitalocean.git",
+                            "plugin_name": "foreman-digitalocean"
```

Comment:
I think this needs to be underscore, it's the gem name that will be added to the Gemfile to load it.